docs/basic-features/font-optimization.md

```diff
@@ -26,39 +26,35 @@ Automatically self-host any Google Font. Fonts are included in the deployment an
 
 Import the font you would like to use from `@next/font/google` as a function. We recommend using [**variable fonts**](https://fonts.google.com/variablefonts) for the best performance and flexibility.
 
-```jsx:app/layout.tsx
+```js:pages/index.js
 import { Inter } from '@next/font/google';
 
 // If loading a variable font, you don't need to specify the font weight
 const inter = Inter();
 
-export default function RootLayout({ children }: {
-  children: React.ReactNode;
-}) {
+export default function Home() {
   return (
-    <html lang="en" className={inter.className}>
```

Comment:
I think we should start off by showing an example using `app`, and then have a second example for `pages`.

Reply:
@hanneslund should we rather use `_document.js`? Something like this
```
export default function Document() {
  return (
    <Html lang="en" className={inter.className}>
      <Head />
      <body>
        <Main />
        <NextScript />
      </body>
    </Html>
  )
}
```
Question about the above and also about using `<html lang="en" className={inter.className}>` in `layout.js` - when you do that, does the default family of any text become `Inter`?

Reply:
> should we rather use _document.js? Something like this

CSS imports doesn't work in `_document`, so not `@next/font` as well. See https://github.com/vercel/next.js/discussions/41849#discussioncomment-3974155

> does the default family of any text become Inter?

Yeah it basically adds a class to the html tag like this:
```tsx
<html className="the-classname-from-inter-call">
```
```css
.the-classname-from-inter-call {
  font-family: "Inter", "Generated Inter Fallback";
}
```

Reply:
@hanneslund Nice. Working on an update. Is there a specific use case for using the wrapper versus inside `<head>`

Reply:
Updated

Reply:
Sorry for being unclear. Meant start by showing with app/layout, then second example with pages/_app.